ebuild-maintenance/general/text.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<guide self="ebuild-maintenance/general/">
+<chapter>
+<title>Maintaining your package</title>
+
+<section>
+<title>General responsibilities</title>
+<body>
+
+<note>
+Gentoo's Quality Assurance (QA) team also maintains a
+<uri link="https://projects.gentoo.org/qa/policy-guide/">policy guide</uri>,
+which contains useful advice on ebuild structure and behaviour.
+</note>
+  
+<p>
+You should always use the latest <uri link="::ebuild-writing/eapi/">EAPI</uri>.
+The main exception is when an eclass you consume does not yet support it.
+You should file a bug or otherwise contact the eclass maintainer to enquire
+about its status with respect to the new EAPI.
+
+<e>Rationale: Newer EAPIs add improvements which are often user-visible,
+e.g. EAPI 6's eapply_user and EAPI 7's improved cross-compile support.</e>
+</p>
+
+<p>
+Check regularly for new bugs on
+<uri link="https://bugs.gentoo.org/">bugs.gentoo.org</uri>.
+
+This includes CI/tinderbox style bugs, which are found by testing the latest
+ebuilds (by commit) and random packages, respectively.
+
+Note that <uri link="https://packages.gentoo.org/">packages.gentoo.org</uri>
+has an experimental Bugs listing per maintainer (see below) but it is
+not yet complete.
+</p>
+
+<p>
+Address CI warnings for your packages. These are generated by pkgcheck.
+You can run <c>pkgcheck scan</c> on your packages to detect most of these,
+but some only become visible when running the scans on the whole tree
+(e.g. broken dependency graph).
+
+Please note that repoman, while advisable to use, at present detects only
+a subset of the issues that pkgcheck does. Given pkgcheck powers Gentoo's CI,
+it is wise to make use of pkgcheck or both tools.
+
+You can view the current full CI report
+<uri link="http://qa-reports.gentoo.org/output/gentoo-ci/output.html">
+here</uri>.
+
+The CI bot should automatically email you if you are responsible for new
+warnings, but you should still check regularly for existing warnings.
+The 'croaker' IRC bot will also mention critical warnings in the #gentoo-dev IRC
+channel to allow tree consistency to be repaired ASAP.
+
+Ask a fellow developer if you're unsure of the meaning or how to fix a warning.
+If in doubt about who to ask, head to the #gentoo-dev or #gentoo-qa IRC channels.
```

Comment:
This is the devmanual, but nondevelopers often use it too, so it might be worth mentioning channels that don't have +m?